main/src/ca/uwaterloo/flix/runtime/Solver.scala

```diff
@@ -438,8 +438,8 @@ class Solver(val root: ExecutableAst.Root, options: Options) {
           evalCross(rule, xs, newRow, interp)
         }
       }
-    case (p: Predicate.Body.Negative) :: xs =>
-      throw InternalRuntimeException("Negated predicates not yet supported")
+    case (p: Predicate.Body.Negative) :: xs => ()
+      //throw InternalRuntimeException("Negated predicates not yet supported")
```

Comment:
Yeah, I will have to fix it. It might take a while which means you won't be able to fully test.